src/app/models/body-type-enum.ts

```diff
@@ -0,0 +1,69 @@
+// The types and codes need to be lowercase for the API.
+
+import { MultiOptions } from '@forms/models/options.model';
+
+export enum BodyTypeDescription {
+  ARTICULATED = 'articulated',
+  BOX = 'box',
+  DOUBLE_DECKER = 'double decker',
+  FLAT = 'flat',
+  OTHER = 'other',
+  PETROL_OR_OIL_TANKER = 'petrol/oil tanker',
+  REFUSE = 'refuse',
+  REFRIGERATED = 'refrigerated',
+  SINGLE_DECKER = 'single decker',
+  SKELETAL = 'skeletal',
+  SKIP_LOADER = 'skip loader',
+  TIPPER = 'tipper',
+}
+
+export function getBodyTypesAsOptions(): MultiOptions {
```

Comment:
We have a function for this here https://github.com/dvsa/cvs-app-vtm/blob/develop/src/app/forms/utils/enum-map.ts

Reply:
Many thanks, using this instead now.